app/web/src/features/dashboard/MyEvents.tsx

```diff
@@ -10,12 +10,11 @@ import Button from "components/Button";
 import HorizontalScroller from "components/HorizontalScroller";
```

Comment:
(Comment not specific to this line, more generally for this file)

The CI pipeline mentions a failing test because `SEE_MORE_EVENTS_LABEL` no longer exists, so I think you'll need to update the tests for this file too (i.e. `MyEvents.test.tsx`)

Reply:
Hi, Darren! Sorry for the delay to answer. I don't know why `yarn test` wasn't able to catch this error, but `yarn test-ci` did the job. I fixed the issue (I hadn't change this file because it was in a another feature folder) and I will submit the code soon again for review. Thank you very much!

Reply:
Sorry, I was logged in to my corporate account when I added my last comments here, haha

Reply:
Fixed. Thanks a lot for the heads up. I ran `yarn lint` but it didn't catch the lint error. Only `yarn test-ci` did the job. Maybe I'll investigate this or add this info to the README later.